### PR TITLE
Isp driver fixes

### PR DESCRIPTION
**Pull request by naushir**
Fixed based on upstream review feedback comments.

### CODE REVIEW

**Comment by naushir** [comment]
@6by9, can you review please?

File: drivers/staging/vc04_services/bcm2835-isp/bcm2835-v4l2-isp.c [hunk 1]
```diff
@@ -1006,15 +1006,29 @@ static int bcm2835_isp_node_s_selection(struct file *file, void *fh,
...
+				      node->q_data.height - s->r.top), MIN_DIM);
+	} else if (s->target == V4L2_SEL_TGT_CROP_DEFAULT) {
+		/* Default (i.e. no) crop window. */
+		s->r.left = 0;
+		s->r.top = 0;
+		s->r.width = node->q_data.width;
+		s->r.height = node->q_data.height;
+	} else {
+		return -EINVAL;
+	}
```
Comment (6by9) [inline drivers/staging/vc04_services/bcm2835-isp/bcm2835-v4l2-isp.c:1034]: switch case?

File: drivers/staging/vc04_services/bcm2835-isp/bcm2835-v4l2-isp.c [hunk 2]
```diff
@@ -1031,29 +1045,34 @@ static int bcm2835_isp_node_g_selection(struct file *file, void *fh,
...
+		s->r.width = node->q_data.width;
+		s->r.height = node->q_data.height;
+	} else  {
 		return -EINVAL;
-
-	s->r.left = crop.rect.x;
-	s->r.top = crop.rect.y;
-	s->r.width = crop.rect.width;
-	s->r.height = crop.rect.height;
+	}
```
Comment (6by9) [inline drivers/staging/vc04_services/bcm2835-isp/bcm2835-v4l2-isp.c:1080]: And again, switch / case

**Comment by 6by9** [comment]
LGTM

**Pull request merged**